app/app.js

```diff
@@ -1,6 +1,6 @@
 const compression = require('compression');
 const config = require('config');
-const cors = require('cors')
+const cors = require('cors');
```

Comment:
We may not need this anymore - we just needed cors as a shim since we were accessing it directly with a stateless frontend. Something for consideration as to whether we want to always allow this going forwards or not (and if yes, backporting to CHES).

Reply:
Yeah will analyze what best-practice for allowing CORS is later...